book_maker/loader/epub_loader.py

```
@@ -68,26 +74,79 @@ def __init__(
+                    if hasattr(it, 'id') and it.id is None:
+                        it.id = 'monkey_patch_dummy'
+                    if hasattr(it, 'children'):
+                        _fix_recursively(it.children)
+                    elif isinstance(it, (list, tuple)):
+                        _fix_recursively(it)
+        
+            if obj.book.toc:
+                try:
+                    _fix_recursively(obj.book.toc)
+                except Exception:
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)